src/pages/StartPage.tsx

```diff
@@ -23,27 +14,22 @@ const StartPage = () => {
   return (
     <Main>
       <MainChild center>
-        {seal ? (
-          <Seal aria-hidden="true" dangerouslySetInnerHTML={{ __html: seal }} />
-        ) : sealURL ? (
-          <Seal aria-hidden="true">
-            <SealImage alt="" src={sealURL} />
-          </Seal>
-        ) : (
-          <React.Fragment />
-        )}
+        <Seal seal={seal} sealURL={sealURL} />
         <Prose textCenter>
           <h1 aria-label={`${title}.`}>{title}</h1>
           <p aria-hidden="true">
             {date}
             <br />
             {county.name}, {state}
-            <br />
-            Ballot Style {ballotStyleId}
-            <br />
-            Precinct {precinctId}
           </p>
+          <hr />
+          <h2>
+            Precinct: {precinctId}
```

Comment:
nit: I was wondering if we maybe should display the precinct name while we're at it. Kind of nicer. Not a blocker.

Reply:
Done.